packages/l/layui.json

```
@@ -22,9 +22,7 @@
       {
         "basePath": "dist",
         "files": [
-          "*.js?(.map)",
-          "css/**/*.css",
-          "font/*.@(eot|ttf|svg|woff|woff2)"
+          "**/*"
```

Generally speaking, we do not allow folks to add open glob-stars in new PRs, as it can lead to files be included that shouldn't be on the CDN. if you're just looking for CSS source maps to be added, please just add that to the existing config, instead of blindly ripping it all out and replacing it with a glob-star.

```suggestion
          "*.js?(.map)",
          "css/**/*.css(.map)",
          "font/*.@(eot|ttf|svg|woff|woff2)"
```